src/migrations/20211216025236_song_guess_count_to_song_metadata.js

```diff
@@ -0,0 +1,31 @@
+exports.up = function (knex) {
+    return knex.schema
+        .renameTable("song_guess_count", "song_metadata")
+        .table("song_metadata", (table) => {
+            table.renameColumn("correct_guesses", "correct_guesses_legacy");
```

Comment:
Don't we need to undo this rename in the exports.down?

Reply:
It falls back to the existing columns (correct_guesses and rounds_played will already exist)